images/builder/runner.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
```

Comment:
Personal opinion: I'd drop the `.sh` part from the filename, or if we're keeping it change it to `.bash`.

Reply:
I will drop the `.sh`